doc/release-notes-17578.md

```diff
@@ -0,0 +1,8 @@
+Deprecated or removed RPCs
+--------------------------
+
+- The `getaddressinfo` RPC `labels` field now returns an array of label name
+  strings. Previously, it returned an array of JSON objects containing `name` and
+  `purpose` key/value pairs, which is now deprecated and will be removed in
+  0.21. To re-enable the previous behavior, launch bitcoind with
```

Comment:
nit, just `launch with -deprecatedrpc...`.

Reply:
Thanks; will remove "bitcoind" here if need to retouch this PR, otherwise will change it in one of the follow-on PRs like #17585 or in the wiki.

Reply:
Sure, just to clarify, I'd drop `bitcoind` because the argument also applies to `bitcoin-qt`.

Reply:
Yes, agreed.

Reply:
Done in #17585